examples/experimental/dagster-airlift/README.md

```diff
@@ -1,26 +1,174 @@
-# dagster-airlift
+# Airlift
 
-The goal of Airlift is to accelerate, lower the cost, and reduce the risk of migrating to Dagster from Airflow. The tool allows you to:
+Airlift is a toolkit for observing Airflow instances from within Dagster and for accelerating the migration of Airflow DAGs to Dagster assets.
 
-- Visualize and observe your existing Airflow DAGs within Dagster with no code changes in Airflow and few lines of code in your Dagster deployment.
-- Model the assets produced by your Airflow instance prior to migration in Dagster (e.g. make your dbt models appear in Dagster while they are orchestrated by Airflow).
-- Enable incremental migration of your Airflow tasks one-by-one, in any order. While the migration is happening, your Airflow DAG structures remain untouched from the standpoint of the observer.
+## Goals
 
-Tl;dr With Airlift, Dagster first becomes your “single pane of glass” for all orchestration and then acts as a toolkit to accelerate a low-risk migration process. Another design partner we are talking to describes this as “changing the oil while driving the car” which I think is a memorable analogy.
+- Observe Airflow DAGs and their execution history with no changes to Airflow code
```

Comment:
It might be worth adding something like "in order to trigger work in Dagster after Airflow DAGs complete"

Reply:
I think that will be confusing because that implies that is the only reason why you would use this feature.

Reply:
Yeah agree that it doesn't make sense to frame it as the only reason to have observability.

Main comment is that I had thought that was one of the main goals and was surprised not to see it mentioned here.

Reply:
I'm going to hold off on that until we add a code example of doing that. I think it should be its own section.